loader/s3_test.go

```diff
@@ -0,0 +1,57 @@
+package loader
```

Comment:
See my comment for `processor_test.go`, but all of these tests could probably be grouped together and only run if the `s3cfg.toml` file is present. Then, the test cases won't just fail if there is no config supplied. Also see the comment about maybe using the package name `loader_test` instead of `loader` for the testing packages.

Reply:
I like this idea a lot--will implement :)